web/src/main/ui/app/components/flows-new/edit-flow/ingest/ui/ingest-ui.component.ts

```diff
@@ -127,7 +151,82 @@ export class IngestUiComponent implements OnInit{
     }
   }
 
+  buildMlcpOptions(): Array<any> {
+    const options: Array<any> = [];
+
+    this.mlcp = {};
+    this.addMlcpOption(options, 'import', null, false, false);
+    this.addMlcpOption(options, 'mode', 'local', false, true);
+
+    const host = this.envService.settings.host;
+    const port = this.envService.settings.stagingPort;
+    const username = this.envService.settings.mlUsername;
+    let input_file_path = this.step.fileLocations.inputFilePath;
+    let input_file_type = this.step.fileLocations.inputFileType;
+    let document_type = this.step.options.outputFormat.toLowerCase();
```

Comment:
The value for input_file_type for mlcp is not the same as the one chosen from the dropdown. Please check the link https://docs.marklogic.com/guide/mlcp/import#id_71112

Reply:
This is Fixed. Please review now.